test/integration/test-extensions-loading.js

```diff
@@ -21,8 +21,6 @@ function checkElementUpgrade(element) {
   expect(element).to.have.class('-amp-element');
   expect(element).to.have.class('-amp-layout-responsive');
   expect(element).to.have.class('-amp-layout-size-defined');
-  expect(element).to.not.have.class('-amp-notbuilt');
```

Comment:
so removing this works @mkhatib ?
Yes since after the change we don't build directly  after upgrade, we can't expect element to be built here. if we dispatch event like `amp:built` later, we can introduce integration test that test all element gets built correctly. (If necessary)